packages/calypso-e2e/src/lib/pages/woocommerce-landing-page.ts

```diff
@@ -0,0 +1,38 @@
+import { Page } from 'playwright';
+
+export const selectors = {
+	start: '.woocommerce .empty-content button:text("Start a new store")',
+	installer: '.is-woocommerce-install',
```

Comment:
For the `installer` and `supportDialog`:

These selectors are just fine, but my personal design approach, especially when I'm doing validation that something actually launched or appeared, is to try to validate some text content within the shells that launch.

So for example, for `supportDialog`, you could do `'text=Introduction to WooCommerce'`.

The reason I generally prefer that is two fold...
1. We try to favor ["user-facing" selectors over things like classes for stability over time](https://playwright.dev/docs/selectors#prioritize-user-facing-attributes).
2. It makes sure there's no WSOD-related errors on the site. I.e., it makes sure the actual content that user's expect is visible within the shell that launches!

100% optional, just wanted to throw out the option!

Reply:
Ah, interesting! Sounds like the right call in this case 👍